src/messages/components/LatestMessages.vue

```diff
@@ -19,6 +19,11 @@
         :label="$t('CONVERSATION.REPLIES')"
         :count="unseenThreadsCount > 9 ? '9+' : unseenThreadsCount"
       />
+      <QRouteTab
+        :title="$t('GROUP.SETTINGS')"
+        icon="fas fa-cog"
+        :to="{ name: 'settings', hash: '#notifications' }"
+      />
```

Comment:
I got a bit confused here, I think because I read "GROUP.SETTINGS" but this is not group settings page it goes to, it's user settings, but also the specific part is notifications.

Hmmm, I have a stronger feeling to actually seperate out this user page into a range of subpages (profile, account, notifications).

But maybe this is a reasonable intermediate solution.

Reply:
... but perhaps a different icon/label to make it clearer that it's the notification settings for the messages?

Reply:
Yes, it can be confusing looking at the code. It's the string I found that would display "Settings" for the user. I found another string named SETTINGS one level up but could not make it work.

As for the icon solution, there is `fas fa-user-cog`, but from the context I prefer having the cog icon only. Another solution would be to stack icons (like envelope and cog). I'll see how it looks